packages/react-components/src/components/datatree-output-component.tsx

```diff
@@ -128,4 +128,7 @@ export class DataTreeOutputComponent extends AbstractOutputComponent<AbstractOut
         // fix Warning: Can't perform a React state update on an unmounted component
         this.setState = (_state, _callback) => undefined;
     }
+    setFocus(): void {
+        return;
```

Comment:
Something should be done to allow data trees so scroll into view even if there is no need for keyboard focus.